memory.go

```diff
@@ -11,9 +11,19 @@ import (
 	"math"
 )
 
+type MemoryBank struct {
+	Name         string `json:"name"`
```

Comment:
Let's go ahead and remove the `Name` field. I don't think it adds anything over the `Label` and `Location` fields.